astropy/table/tests/test_table.py

```diff
@@ -1018,6 +1018,18 @@ def test_rename_by_attr(self, table_types):
         assert np.all(t['c'] == np.array([1, 2, 3]))
         assert np.all(t['a'] == np.array([4, 5, 6]))
 
+    def test_rename_columns(self, table_types):
+        self._setup(table_types)
+        t = table_types.Table([self.a, self.b, self.c])
+        t.rename_columns(('a', 'b', 'c'), ('aa', 'bb', 'cc'))
+        assert t.columns.keys() == ['aa', 'bb', 'cc']
+        t.rename_columns(['bb', 'cc'], ['b', 'c'])
+        assert t.columns.keys() == ['aa', 'b', 'c']
+        with pytest.raises(TypeError):
+            t.rename_columns(('aa'))
```

Comment:
This is giving a `TypeError` because of the wrong number of arguments.  That is not really a useful or necessary test here, and I think you mean to do something like `t.rename_columns('aa', ['a'])` to demonstrate that giving an incorrectly typed argument (not a list or tuple) raises a `TypeError`.